packages/core/lib/services/wallet/service.js

```diff
@@ -275,6 +276,27 @@ export class WalletService {
 
     return this.wallet.getStorageDocument({id}).then(doc => doc.content);
   }
+
+  async getDocumentsFromEncryptedWallet(params) {
```

Comment:
we're good with errors bubbling out of this function?

Reply:
Yes we are. The plan is to allow the caller handle the exception how they chose to.